reana_workflow_controller/tasks.py

```diff
@@ -99,3 +105,20 @@ def _update_job_progress(workflow_uuid, msg):
                     run_job.run_id = current_run.id_
                     run_job.job_id = job_id
                     Session.add(run_job)
+
+
+def _update_job_cache(msg):
+    """Update caching information for finished job."""
+    cmd = msg['caching_info']['job_spec']['cmd']
+    clean_cmd = cmd.split(';')[1]
```

Comment:
This is a "hack" to clean the commnands from `r-w-e-serial`, that have prepended: `cd <path-to-workspace>` which includes a unique id, making the command hash different each time.

Reply:
👍  we can add a comment so we remember